app.css

```diff
@@ -0,0 +1,46 @@
+body {
+  direction: rtl;
+  padding: 0px;
+  margin: 0px;
+}
+
+.container {
+    width: 960px;
+    margin: auto;
+}
+
+/*navbar*/
+
+.headWrapper{
+  height: 60px;
+  border-bottom: 2px solid;
+  background-color: #373d46;
+  color: #4cc5e2;
+}
+
+.headWrapper h1 {
+    float:right;
+}
+
+.headWrapper header {
+    height:60px;
+}
+
+.headWrapper header ul {
+  margin: auto;
+  width: 500px;
```

Comment:
Not sure why are you giving this div a width. Remember it should be left aligned inside the 960px container. Try again...